hw/ip/lc_ctrl/data/lc_ctrl_testplan.hjson

```diff
@@ -85,7 +85,7 @@
             - Check if lc_state moves to escalation state.
             '''
       milestone: V2
-      tests: []
+      tests: ["lc_ctrl_state_failure"]
```

Comment:
Just a heads up that design will change the FSM to a prim_FSM and we will add another standardized sequence to inject FSM error, which might overlap with this test's checking. But I think this test is unique and definitely needed because we are testing that lc_state is moving to escalation state.
@weicaiyang FYI.